docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -310,6 +312,25 @@ Uninstall Alluxio as follows:
 $ helm delete alluxio
 ```
 
+#### Format Journal
+
+The helm chart contains one Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+template that can be used for formatting the Alluxio journal. The template contains one Job for each Alluxio master.
+Each Job runs `alluxio formatJournal` and formats the journal for that master.
+
+You can manually trigger the journal formatting by applying the Job template.
+```console
+# Generate the YAML definition for the journal formatting Job(s)
+$ helm template --name alluxio helm-chart/alluxio/ --set journal.format.runFormat=true -x templates/job/format-journal-job.yaml -f ./config.yaml > alluxio-format-journal-job.yaml
```

Comment:
instead i would suggest `helm install` with the option `journal.format.runFormat=true` for formatting explicitly before starting the masters.

Reply:
My concern is the Job contending with master on the journal PV if the PV accessMode is `ReadWriteOnce`.

Reply:
To ensure EITHER the jobs OR the masters are running, I would go with using the `.Values.journal.format.runFormat` to decide whether the `master-statefulset.yaml` should be empty. And use `helm upgrade` to do the switch. What do you think?

Reply:
looks like there is an order of deployment: https://github.com/helm/helm/blob/484d43913f97292648c867b56768775a55e4bba6/pkg/releaseutil/kind_sorter.go#L27 but it is the reverse of what we want, statefulset is first and then job

so in this case i'm good with your suggestion to switch, but i would like to only run the format job and nothing else (not even worker install). a simple if over master-statefulset seems hard to maintain. might want to see if helm allows to specify resource dependencies before we go ahead with the switch approach.

Reply:
@jiacheliu3 how about format as an init container instead?

Reply:
@madanadit I think format as init container is a better idea overall. 

In that case i will only include 2.2-related doc in this change and add blocks regarding the `iniContainer` approach later, after format has been made init container.

Reply:
@madanadit 
I found myself wrong with `ReadWriteOnly`. This `Only` means only one NODE.
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes

Therefore the Job can be created and completed even if the master is there. What this means is, although formatting the journal when the master is active is not the best idea, it can work.

Reply:
I'm okay with this for now, but you could file an issues to track replacement of the format job with init container (if not there already)

Reply:
https://github.com/Alluxio/alluxio/issues/11141 created to address this.